### DISCUSSION TITLE
Clear Filter without Command

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#1960] [Q&A]
Hey,

is it possible to clear the filter within a datagrid without having to use a CommandCol? Because I just want to set it outside of the grid more or less when pressing a button to toggle the filters. So if I hide all filters their searchvalue will get reset.

**Comment by stsrki** [comment]
It's not possible at the moment. The API to control the DataGrid is still in the planning stage and it should be done for the 0.9.4 milestone.
